.omeroci/app-deps

```diff
@@ -3,4 +3,4 @@
 set -e
 set -u
 yum -y install https://rpm.nodesource.com/pub_6.x/el/7/x86_64/nodesource-release-el7-1.noarch.rpm
-yum -y install npm
+yum -y install nodejs
```

Comment:
This is a new problem. This matches what is needed on iviewer. We may want in the feature to install a newer version of nodejs